backend/entities/feedback_entity.py

```diff
@@ -0,0 +1,100 @@
+"""Definition of SQLAlchemy table-backed object mapping entity for Feedback."""
+
+from sqlalchemy import Column, Integer, String, Boolean, ForeignKey
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from backend.models.feedback import Feedback
+from backend.models.feedback_details import FeedbackDetails
+from .entity_base import EntityBase
+from typing import Self
+from ..models.user import User
+
+
+__authors__ = ["Matt Vu"]
+__copyright__ = "Copyright 2023"
+__license__ = "MIT"
+
+
+class FeedbackEntity(EntityBase):
+    """Serves as the database model schema defining the shape of the `Feedback` table"""
+
+    # Name for the feedback table in the PostgreSQL database
+    __tablename__ = "feedback"
+
+    # Unique ID for the feedback
+    id: Mapped[int] = mapped_column(Integer, primary_key=True, autoincrement=True)
+    # User who submitted the feedback ticket
+    user_id = mapped_column(Integer, ForeignKey("user.id"))
+    user = relationship("UserEntity")
+    # Description of the feedback
+    description: Mapped[str] = mapped_column(String)
+    # Whether the feedback has been resolved or not
+    has_resolved: Mapped[bool] = mapped_column(Boolean)
```

Comment:
Nit: I think Kris has moved us to a convention where flag fields do not include an `is_` or a `has_` prefix! `resolved` works perfectly here as the field name.